OpenOversight/app/models.py

```diff
@@ -101,6 +101,7 @@ class Officer(db.Model):
     employment_date = db.Column(db.Date, index=True, unique=False, nullable=True)
     birth_year = db.Column(db.Integer, index=True, unique=False, nullable=True)
     assignments = db.relationship('Assignment', backref='officer', lazy='dynamic')
+    assignments_lazy = db.relationship('Assignment')
```

Comment:
This line is necessary, to be able to use the `joinedload` on assignments, to load all assignments for all relevant officers in one go instead of a separate query for every single officer (known as N+1 problem)

Reply:
:+1: